_templates/playground/new/prompt.js

```diff
@@ -0,0 +1,32 @@
+// see types of prompts:
+// https://github.com/enquirer/enquirer/tree/master/examples
+//
+module.exports = [
+  {
+    type: 'input',
+    name: 'name',
+    message: 'Which component is this playground for?',
+    initial: 'ion-button',
+  },
+  {
+    type: 'input',
+    name: 'path',
```

Comment:
Should I be able to enter spaces for the directories?

```
✔ Which component is this playground for? ·  
✔ What should the playground path be? ·  
✔ Select the Ionic Framework version for the playground · 7
✔ Do you want CSS scaffolded? · No / Yes

Loaded templates: _templates
       added: static/usage/v7/ / /angular.md
       added: static/usage/v7/ / /demo.html
       added: static/usage/v7/ / /index.md
       added: static/usage/v7/ / /javascript.md
       added: static/usage/v7/ / /react.md
       added: static/usage/v7/ / /vue.md
       shell: echo `
Generation complete! To use this new component in a docs markdown file, include
the following:

## 

import  from '@site/static/usage/v7/ / /index.md';

< />
`
```

Reply:
Is this something we expect to need often?

As far as this and a couple other comments go, I wasn't really worrying about edge cases or error handling for now since this is just a tool to generate documentation. I talked about that some in my design doc, which will be up for review shortly.